Sources/LiveKit/Types/Options/RoomOptions.swift

```diff
@@ -72,6 +72,10 @@ public final class RoomOptions: NSObject, Sendable {
     override public init() {
         defaultCameraCaptureOptions = CameraCaptureOptions()
         defaultScreenShareCaptureOptions = ScreenShareCaptureOptions()
+        #if os(iOS)
+            .settingUseBroadcastExtension(BroadcastBundleInfo.hasExtension)
+        #endif
```

Comment:
this would be better set in the default constructor for ScreenshareCaptureOptions than in the default constructor for Room only